### PR TITLE
Story/vspc 77

### PR DESCRIPTION
**Pull request by prjadhav14**

### CODE REVIEW

File: vspace/src/main/webapp/WEB-INF/views/exhibition/space.jsp [hunk 1]
```diff
@@ -9,15 +9,27 @@
 <script>
 //# sourceURL=click.js
 function openNav(){
-    document.getElementById("mySidenav").style.width = "240px";
-    document.getElementById("mySidenav").style.height = ${display.height}+"px"
+    if (window.matchMedia('(max-width: 800px)').matches){
```
Comment (jdamerow) [inline vspace/src/main/webapp/WEB-INF/views/exhibition/space.jsp:12]: let's put the string `(max-width: 800px)` and reuse that throughout the file

**Comment by jdamerow** [comment]
Jenkins, test this please.

**Comment by jdamerow** [comment]
the image doesn't seem to change size when I change the window size

**Comment by jdamerow** [comment]
I think you need to prevent the image from being resized until a certain point is reached. Right now the links don't get recalculated until you reach that point, so that they get misaligned:
wrong:
image
correct:
image

**Comment by prjadhav14** [comment]
> I think you need to prevent the image from being resized until a certain point is reached. Right now the links don't get recalculated until you reach that point, so that they get misaligned:
> wrong:
> image
> correct:
> image

What is screen resolution for the wrong image? need to replicate the issue in local environment.

**Comment by jdamerow** [comment]
I just changed my browser window size.

**Comment by prjadhav14** [comment]
> I just changed my browser window size.

Ok, by using zoom in and zoom out or toggling the screen?

**Comment by jdamerow** [comment]
Jenkins, test this please.

**Comment by jdamerow** [comment]
The icons have off now:
image

**Comment by jdamerow** [comment]
Jenkins, test this please.

File: vspace/src/main/webapp/WEB-INF/views/exhibition/space.jsp [hunk 2]
```diff
@@ -7,17 +7,29 @@
     href="<c:url value="/resources/extra/Home.css" />">
 
 <script>
-//# sourceURL=click.js
```
[the remark this answers is not in the document]
Comment (prjadhav14) [reply vspace/src/main/webapp/WEB-INF/views/exhibition/space.jsp:10]: I thought it is not useful as is commented code. Should I add that line back?

**Pull request merged**